SharedCryptoPkg/Package/SharedCryptoPkgDxe.inf

```diff
@@ -0,0 +1,61 @@
+## @file
+#  Crypto Bin Driver package
+#
+# Copyright (c) 2019, Microsoft Corporation
+#
+# All rights reserved.
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+# 1. Redistributions of source code must retain the above copyright notice,
+# this list of conditions and the following disclaimer.
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+# this list of conditions and the following disclaimer in the documentation
+# and/or other materials provided with the distribution.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+# ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
+# IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+# INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+# BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+# DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+# LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE
+# OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
+# ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+##
+
+[Defines]
+  INF_VERSION                    = 0x0001001B
+  BASE_NAME                      = SharedCryptoDriverDxe
+  FILE_GUID                      = BDFC3C26-73AF-4A14-B861-1B6922EE0C46
+  MODULE_TYPE                    = DXE_DRIVER
+  VERSION_STRING                 = 1.0
+  ENTRY_POINT                    = SharedCryptoDxeEntry
+
+#
+# The following information is for reference only and not required by the build tools.
+#
+#  VALID_ARCHITECTURES           = IA32 X64 AARCH64
+#
+
+# these binaries are resolved by the Nuget Package
+# if they are not found, check there
+
+[Binaries]
+  DXE_DEPEX|Mu-SharedCrypto_extdep/X64/SharedCryptoDxe.depex|*
+
+[Binaries.X64]
+  PE32|Mu-SharedCrypto_extdep/X64/SharedCryptoDxe.efi|*
```

Comment:
Where is this path?

It may be better to merge the source content of this PR into the CryptoPkg, and to build releases of the CryptoPkg into a binary release of the CryptoPkg that contains the PEIM, DXE Driver, and DXE SMM Driver.

Reply:
The extdep resolved into SharedCryptoPkg/Package. You can see the dependency in SharedCryptoPkg\Package\SharedCrypto_ext_dep.json. I like that idea of moving this into CryptoPkg- makes it simpler as the header is in the same place as the wrapper

Reply:
I think for now we'll have this live in Mu_plus and work on getting this upstreamed to Tianocore in CryptoPkg.